packages/app-store/apps.browser.generated.tsx

```diff
@@ -3,6 +3,7 @@
     Don't modify this file manually.
 **/
 import dynamic from "next/dynamic";
+
```

Comment:
This is due to new prettier config according to hariom. So a change that should only occur once, then it's on main and shouldn't happen anymore.